portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,56 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+
+			DMDocument.enableActionsMenuOnPortlet();
+
```

Comment:
Hey, do you think enabling this action menu is necessary?

Reply:
It is, because the widget doesn't enable actions like Add Entry or Edit by default, so I have to enable them in order to interact with the DMs in the page instead of going to the Product Menu. That's one of the solutions I found by searching around other testcases.

I'll try to find a better solution, though.

Reply:
Okay, I'll make the changes and send them here. Thanks!